docs/content/en/docs/yaml-crd-ref/task.md

```diff
@@ -0,0 +1,206 @@
+---
+title: KeptnTask
+description: Define a run of a KeptnTaskDefinition
+weight: 85
+---
+
+Keptn populates the `KeptnTask` resource
+for tasks that deploy software on Kubernetes.
+When using Keptn to deploy software built on a virtual machine,
```

Comment:
Perhaps just say "When using Keptn to run tasks for software that is deployed outside of kubernetes". This method would work for things deployed on anything from mainframes to VMs to serverless etc.

Reply:
Yes, this is much better -- done.